src/Strategy/NullableStrategy.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-hydrator for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-hydrator/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-hydrator/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Hydrator\Strategy;
+
+class NullableStrategy implements StrategyInterface
+{
+    /**
+     * @var StrategyInterface
+     */
+    private $strategy;
+
+    /**
+     * @param StrategyInterface $strategy
+     */
```

Comment:
Can be remove the because the same information is already included in the method signature.